.github/workflows/release.yml

```diff
@@ -70,13 +42,14 @@ jobs:
       - name: Deploy artifacts
         id: deploy
         if: github.ref == 'refs/heads/main'
-        run: ./deploy.sh
+        run: |
+          ./gradlew publishToSonatype closeAndReleaseStagingRepository --stacktrace
 
       - name: Bump version and push tag
         uses: anothrNick/github-tag-action@1.67.0
         if: steps.deploy.outputs.RELEASE == 'true' && github.ref == 'refs/heads/main'
```

Comment:
if we delete the deploy.sh file, who is going to set this RELEASE=true now?

Same for the "Deploy Javadoc" step below